### DISCUSSION TITLE
Breadcrumb appearance questions

### DISCUSSION THREAD

**Discussion by pkothare** [microsoft/fluentui-blazor#1989] [Q&A] [answered]
Typical implementations of a breadcrumb allow you to distinguish between ancestor breadcrumb items and the leaf breadcrumb by doing a visual scan using color, usually because the ancestors are hyperlinks. For example:

From the FAST site: https://explore.fast.design/components/fast-breadcrumb

![image](https://github.com/microsoft/fluentui-blazor/assets/3177809/d71a2634-c75a-4d93-84f1-63b61bd2417b)

From GitHub:

![image](https://github.com/microsoft/fluentui-blazor/assets/3177809/1584865b-186d-4b39-bc12-4dd83905ffc1)

Is there any way to achieve the same outcome with the `FluentBreadcrumbItem` without having to override shadow dom CSS? Is there a simpler solution? If there is, I wasn't able to infer it from the [Breadcrumb - demo site](https://www.fluentui-blazor.net/Breadcrumb).

**Comment by vnbaaij** [comment, marked as answer]
You can use the `Class` and/or `Style` parameters to change the appearnace of items anyway you like (Class and Style are available to any component derived from `FluentComponentBase`)

Doing somthing like

**BreadcrumbDefault.razor**
```
<span>
    <FluentBreadcrumb>
        <FluentBreadcrumbItem Class="parent" Href="#">
            Breadcrumb item 1
        </FluentBreadcrumbItem>
        <FluentBreadcrumbItem Class="parent" Href="#">
            Breadcrumb item 2
        </FluentBreadcrumbItem>
        <FluentBreadcrumbItem>Breadcrumb item 3</FluentBreadcrumbItem>
    </FluentBreadcrumb>
</span>
```
and 

**BreadcrumbDefault.razor.css**
```
::deep fluent-breadcrumb-item.parent {
    color: #ff6a00;
}
```

Will give you:
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/1b849aa3-d3ca-42f5-b86b-1790c77b6eaf)
